docs/guide/results/working_with_networks.rst

```diff
@@ -3,8 +3,56 @@
 Working with networks of results
 ================================
 
-issue #803
+After running a **network** of free energy calculations,
+we often want to consider analysing the corresponding network of results.
 
+.. _userguide_MLE:
 
-The :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
+Converting relative results to absolute estimates
+-------------------------------------------------
+
+When a network of relative free energies have been calculated,
+a commonly performed task is to transform these pairwise estimations of **relative** free energies
```

Comment:
Technically, we never obtain actual "free energies" but always "free energy differences" (so we don't get G but always Delta G or Delta Delta G). I know that always sounds very wordy so I'm not sure if it needs to be changed here or if we can leave it as is...